docs/routing/codable-routing.html

```diff
@@ -142,26 +130,36 @@ <h2 class="heading-2"><span class="blue-text">Step 1:</span> Create a file to co
     static var codableStore = [Book]()
     // Write handlers here
 }</code></pre>
-        <p>We have imported the KituraContracts library as it contains the shared type definition for `RequestError` which we will use in the next step.</p>
+        <p>This code contains two sections. The first is the `initializeCodableRoutes` function.
+            This will be run after the application has been initialized and is where we register routes on our router.
+            The second is the `App` extension. This is where we write the logic to handle requests.</p>
+        <p>We also have a `codableStore` where we will store an array of objects. We will use the <a target="_blank" href="https://www.kitura.io/docs/getting-started/style-guide.html"> Book model from the style guide</a>, however you could use any Codable type.</p>
+        <p>Finally, we have imported the KituraContracts library as it contains the shared type definition for `RequestError` which we will use in the next step.</p>
```

Comment:
We should have an add KituraContracts as a dependency link here if we're using KituraContracts.

Reply:
We are not importing KituraContracts since it gets pulled in by Kitura.

Reply:
Why can't we say that explicitly so it's clear to users what we're doing.